### ISSUE TITLE
multiple banks of samples?

### ISSUE THREAD

**Issue by lukeoftheaura** [openAVproductions/openAV-Fabla2#68]
does f2 have support for multiple banks of samples? I assume this is what A B C and D are meant to be, but other than UI colours changing nothing happens when I switch between them.

**Comment by harryhaaren** [comment]
Yes the ABCD buttons in the top left are for banks. The functionality is there - but its currently disabled due to a segfaulting bug in the bank code somewhere - I'd have to spend some time debugging to get to the root cause. Nothing that can't be fixed - its just about time and priorities.
